app/views/crosswalk_issues/find_matches.html.erb

```diff
@@ -29,19 +29,21 @@
             <th>Address</th>
             <th>OPE</th>
             <th>IPE</th>
+            <th>Matching Score</th>
             <th>Actions</th>
             </thead>
             <tbody>
-            <% @ipeds_hd_arr.each do |iped| %>
+            <% @ipeds_hd_arr.each do |i| %>
```

Comment:
Why was this variable changed to a less descriptive name?

Reply:
Ha! I honestly don't remember making that change. Good catch!